source/configure/user-management-configuration-settings.rst

```diff
@@ -105,11 +105,24 @@ To delete a user from your Mattermost deployment, you can deactivate the user's
 .. image:: ../images/deactivate-user.png
   :alt: Deactivate a user in Mattermost using the System Console.
 
-5. If the user is deactivated, select **Activate** to re-activate the user.
+You can re-activate a deactivated user by selecting **Activate**.
 
 .. image:: ../images/activate-user.png
   :alt: Activate a user in Mattermost using the System Console.
 
+What happens to deactivated user integrations?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. important::
+
+  If you deactivate a Mattermost user who has integrations tied to their user account, consider the following consequences and recommendations based on the integration type:
+
+  - **Slash commands** will continue to work after user deactivation. You may want to consider deleting the existing slash command token and creating a new slash command associated with a different user account to decouple sensitive token data from the deactivated user account.
+  - **Outgoing webhooks** will continue to work after user deactivation; however, access to webhook tokens is limited to team admins and system admins only, so the risk of exposing sensitive token data is lower. You may want to consider deleting the existing outgoing webhook token and creating a new outgoing webhook associated with a different user account.
```

Comment:
When the outgoing webhook is created, the to-be-deactivated user who creates the outgoing webhook gets to see the webhook token both as the response to the webhook creation and when they visit the the outgoing integrations. 

So this phrase is not accurate. The creator of the webhook can always see the webhook token:
>  however, access to webhook tokens is limited to team admins and system admins only, so the risk of exposing sensitive token data is lower.

We can suggest that the regenerate the webhook token and also that they check that the deactivated user no longer has access to the callback URLs, as having access would result in the deactivating user receiving the outgoing webhooks.

![Screenshot 2024-06-12 at 12 11 18](https://github.com/mattermost/docs/assets/17114347/7d7699b1-24ef-4efa-96e3-678dbf17a7f2)